packages/web-components/src/switch/switch.stories.ts

```diff
@@ -0,0 +1,77 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import type { Switch as FluentSwitch } from './switch.js';
+import './define.js';
+import { SwitchLabelPosition } from './switch.options.js';
+
+type SwitchStoryArgs = Args & FluentSwitch;
+type SwitchStoryMeta = Meta<SwitchStoryArgs>;
+
+const storyTemplate = html<SwitchStoryArgs>`
+  <div>
+    <fluent-switch
+      ?checked=${x => x.checked}
+      ?disabled=${x => x.disabled}
```

Comment:
Adding a `required` attribute to `disabled` switch results in an error in console (Chrome 109.0.5414.119 on Mac)
cc @chrisdholt 
![image](https://user-images.githubusercontent.com/9615899/216023308-8dcbeec2-b6dc-4144-9c82-c99c340e7ec9.png)

Reply:
@chrisdholt I'm also seeing this on my windows machine in chrome. Any ideas?

Reply:
Can you file a bug please.

Reply:
@chrisdholt https://github.com/microsoft/fast/issues/6635

Reply:
@chrisdholt https://github.com/microsoft/fast/issues/6635